esignet-insurance.properties

```diff
@@ -441,25 +441,31 @@ mosip.esignet.vci.key-values={\
               'display': {{'name': 'Insurance', 'locale': 'en'}},\
               'credentials_supported': {{\
                       'format': 'ldp_vc',\
-                      'id': 'InsuranceCredential', \
+                      'id': 'VehicleCredential', \
```

Comment:
We should not replace this, we should have both Insurance and Vehicle separately as 2 items in the list

Reply:
Lets rename VehicleCredential to VehicleInsuranceCredential everywhere.

Reply:
changes updated.